### ISSUE TITLE
iZettle issues within intelligentPOS (Sweden)

### ISSUE THREAD

**Issue by ONeill117** [iZettle/sdk-ios#125]
account email: orders@hoteletail.com

At location A, I have a standard iZettle terminal. It's perfect and I love it.

Since I can't have two distinct iZettle terminal layouts, I have had to go for intelligentPOS at location B. (My plan is to then roll out intelligentPOS at location C, D, E, ...

I am having REPEATED issues with the iZettle integration to intelligentPOS, which intelligentPOS says is the fault of iZettle, not intelligentPOS.

On 26/01, the iZettle account automatically logged out by itself, meaning we lost sales opportunities (since the staff don't have the iZettle log-in details).
On 14/02, the iZettle account automatically logged out, causing loss of sales as well. 
On 15/02, the intelligentPOS app froze, and after rebooting, the iZettle account was logged out. Here I did a hard reset of everything on the iPad and rebooted everything fully, hoping to eliminate the problems going forward.
On 16/02, the iZettle account froze when taking a payment, I rebooted intelligentPOS (which was working fine), but when taking a payment again, the iZettle account froze. I then had to manually log out of iZettle and log back in (from within the intelligentPOS account).

I have been on the phone back and forth with iZettle and intelligentPOS, and have heard all sorts of legitimate excuses ("there was an iOS log-in bug on version 10.x", "there was a iZettle log-in bug in January", "intelligentPOS can freeze/reset if you have low memory"), so MAYBE I've just had a hump of bad luck and it's over, but I CAN'T afford to have these problems reoccuring, especially not 3 times a week!

1. What can I do to PREVENT these issues from happening in the first place? (Fixing them is no problem, but since I'm not onsite every day, it's possible we lose an entire few day's worth of potential sales.)

2. Is there any way for me to remotely check in on the status of the iPad/intelligentPOS/iZettle account?

3. Why can't I just have 2 iZettle accounts and eliminate the need for intelligentPOS? Location A is working perfectly, but I need a robust solution for location B, C, D, ...

Thanks for any advice/solutions!

**Comment by mattiasjahnke** [comment]
Hi @ONeill117 

Sorry to hear that you've been passed around support departments coming with excuses. We have a great collaboration with intelligentPOS tech department, and we'll make sure to troubleshoot this problem together with them.

1. I can't say if there's anything you can do in order to prevent these problems from happening, since I'm not quite sure what's going on yet.

2. No

3. I don't see why you wouldn't be able to do that. Often times merchants have "subaccounts" to be used on location, tied to an "owner"s account. If you don't need features that only intelligentPOS can offer, and you're happy with iZettles app solution, you could just use that.

Again, sorry to hear that you are experiencing problems, we'll investigate what's going on - your iZettle session should stay active in the intelligentPOS app unless you log out.

**Comment by ONeill117** [comment]
Hi @mattiasjahnke, and thank you for the quick reply! much appreciated.

The beauty of an acquistion (such as of intelligentPOS by iZettle) is that the customer gets the best of both worlds :) The downside is that it can sometimes take many months to get things fully integrated and rolled out. It can also mean that I could end up in a position where both sides' tech teams just blame the other. So it's good to see that you guys are working together!

1. If you need any further details, let me know or we can schedule a call. My intelligentPOS user name is hoteletail.

2. OK :P 

3. Yes I would love to do that. I would love to have multiple, distinct iZettle accounts (the user interface of iZettle is way smoother than intelligentPOS). I need location/terminal A with products x, y, z, and location/terminal B with products v, w, x, for example. My understanding was that with iZettle, you register with 1 company account / org#, and you can have multiple terminals but you HAVE to display the same products at each terminal. The reason I started with intelligentPOS was the ability to have  separate 'terminal layouts' as they're called, so location A has terminal layout 1 and location B has terminal layout 2.
A 'solution' I figured for iZettle was multiple folders, but each terminal/location was LOCKED to a specific folder.

I was actually considering the idea of creating a new company JUST to be able to register a second iZettle account. That's how bad it's gotten! Interested to hear more about your 'subaccount' solution...

In summary, my needs are the following (in order of priority):

1. idiot-proof, stable customer interface (which iZettle provides). I need untrained staff to be able to intuitively take payments.

2. distinct products available at different locations (like intelligentPOS offers)

3. the ability to manage all products/terminals centrally (like iZettle and intelligentPOS both do). This is the least important because I'll happily use a completely separate POS solution at each location, PROVIDED that conditions 1 and 2 are met.

Look forward to hearing your thoughts.

**Comment by mattiasjahnke** [comment]
Let's continue this discussion on e-mail. Can I use orders@hoteletail.com or do you prefer another?

**Comment by ONeill117** [comment]
@mattiasjahnke 

nathan@hoteletail.com works best thanks!
